backend/__tests__/integration/auth/schedule.spec.ts

```diff
@@ -1,53 +1,170 @@
+import { HttpStatus } from '@nestjs/common';
+import { availableSchedule } from '../../../src/global/test-data/schedule-test-data.service';
+import {
+  AvailabilityEnum,
+  ScheduleDTO,
+} from '../../../src/schedule/dto/schedule.dto';
 import { ScheduleController } from '../../../src/schedule/schedule.controller';
+import {
+  createSchedule,
+  testCreateSchedule,
+} from '../fixtures/schedule.fixtures';
+import { requestWithStaff } from '../setup';
 
 describe('schedule.spec.ts - Schedule Controller', () => {
   const scheduleRoute = ScheduleController.SCHEDULE_API_PATH;
-  // eslint-disable-next-line
-  const idPath = scheduleRoute + ScheduleController.ID_API_PATH;
 
   describe('POST /', () => {
-    // eslint-disable-next-line
-    it('should create schedule', () => {});
+    it('should create schedule', async () => {
+      const { availability, startTime, endTime }: ScheduleDTO =
+        await createSchedule(testCreateSchedule);
 
-    // eslint-disable-next-line
-    it('should not create schedule with missing data', () => {});
+      expect(availability).toEqual(testCreateSchedule.availability);
+      expect(startTime).toEqual(testCreateSchedule.startTime.toISOString());
+      expect(endTime).toEqual(testCreateSchedule.endTime.toISOString());
+    });
 
-    // eslint-disable-next-line
-    it('should not create schedule when startTime is after endTime', () => {});
+    it('should not create schedule with missing data', async () => {
+      const createSchedWithoutStartTime = {
+        availability: AvailabilityEnum.AVAILABLE,
+        endTime: new Date('2022-9-3 5:00:00 PM'),
+      };
+      const createSchedWithoutEndTime = {
+        availability: AvailabilityEnum.AVAILABLE,
+        startTime: new Date('2022-9-3 2:00:00 PM'),
+      };
+
+      await requestWithStaff.post(scheduleRoute).expect(HttpStatus.BAD_REQUEST);
+      await requestWithStaff
+        .post(scheduleRoute)
+        .send(createSchedWithoutStartTime)
+        .expect(HttpStatus.BAD_REQUEST);
+      await requestWithStaff
+        .post(scheduleRoute)
+        .send(createSchedWithoutEndTime)
+        .expect(HttpStatus.BAD_REQUEST);
+    });
+
+    it('should not create schedule when startTime is after endTime', async () => {
+      const createStartTimeAfterEndTimeSched = {
+        availability: AvailabilityEnum.AVAILABLE,
+        startTime: new Date('2022-9-3 2:00:00 PM'),
+        endTime: new Date('2022-9-3 1:00:00 PM'),
+      };
```

Comment:
no need to change but we could put a type to this variable. so it would become 
```javascript
const createStartTimeAfterEndTimeSched: ScheduleDTO = ...
```